### PR TITLE
Reset the SAML flow when an exception occurs

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

**Comment by rjkip** [comment]
knot
@DRvanR

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Firewall/SamlListener.php [hunk 1]
```diff
@@ -74,6 +74,17 @@ public function __construct(
 
     public function handle(GetResponseEvent $event)
     {
+        try {
+            $this->tryHandle($event);
```
Comment (DRvanR) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Firewall/SamlListener.php]: `handleEvent`? tryHandle sounds so meh...

**Comment by DRvanR** [comment]
Minor nitpick-of-mehness, otherwise :+1:

**Comment by DRvanR** [comment]
(same can be applied in RA btw, GW not so sure)

**Pull request merged**